crates/speedy-transform/src/web_transform/babel_import.rs

```diff
@@ -44,35 +54,27 @@ pub fn transform_style(
   }
   let mut specifiers_css = vec![];
   let mut specifiers_es = vec![];
-  let mut specifiers_rm_es = vec![];
+  let mut specifiers_rm_es = HashSet::new();
 
   let config = project_config.babel_import.as_ref().unwrap();
 
-  for item in &module.body {
-    let item_index = module.body.iter().position(|citem| citem == item).unwrap();
+  for (item_index, item) in module.body.iter_mut().enumerate() {
     if let ModuleItem::ModuleDecl(ModuleDecl::Import(var)) = item {
       let source = &*var.src.value;
       if let Some(child_config) = config.iter().find(|&c| c.from_source == source) {
-        for specifier in &var.specifiers {
+        let mut rm_specifier = HashSet::new();
+        for (specifier_idx, specifier) in var.specifiers.iter().enumerate() {
           match specifier {
             ImportSpecifier::Named(ref s) => {
-              let imported = match &s.imported {
-                Some(imported) => match &imported {
-                  ModuleExportName::Ident(ident) => Some(ident.sym.to_string()),
-                  ModuleExportName::Str(ident) => Some(ident.value.to_string()),
-                },
-                None => None,
-              };
+              let imported = s.imported.as_ref().and_then(|imported| match imported {
```

Comment:
It's better to use `map` here to avoid manual `Option` wraps

Reply:
fix